packages/firebase_data_connect/firebase_data_connect/example/dataconnect.yaml

```diff
@@ -0,0 +1,11 @@
+location: "us-west2"
```

Comment:
Is there any impact depending on the location of an emulator project?

Reply:
No.